File: 1351/a/a.java
import java.io.*;
import java.math.*;
import java.util.*;
import java.text.*;

public class Main {
    public static void main(String[] args) {
        Scanner cin = new Scanner(new BufferedInputStream(System.in));
        int t;
        t = cin.nextInt();
        while (t-- != 0) {
            int a, b;
            a = cin.nextInt();
            b = cin.nextInt();
            System.out.printf("%d\n", a + b);
        }
    }
}

File: 1351/a/a.kt
import java.io.BufferedInputStream
import java.util.*

    fun main(args: Array<String>) {
        val cin = Scanner(BufferedInputStream(System.`in`))
        var t: Int
        t = cin.nextInt()
        while (t-- != 0) {
            var a: Int
            var b: Int
            a = cin.nextInt()
            b = cin.nextInt()
            System.out.printf("%d\n", a + b)
        }
    }